codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -565,11 +720,18 @@ int incrementStreamWindow(int delta) throws Http2Exception {
             if (streamableDelta != 0) {
                 incrementStreamableBytesForTree(streamableDelta);
             }
+
+            // Check if a writability change has occurred, and save it for processing at some later point.
+            if (!writabilityChanged) {
+                writabilityChanged = previouslyWritable ^ (writableWindow() > 0);
```

Comment:
This seems to be the only place we set `writabilityChanged`.  Should it also be set by `state.writeAllocatedBytes`?